common/source/docs/common-rc-transmitter-flight-mode-configuration.rst

```diff
@@ -155,50 +156,51 @@ systems:
    an even easier way)
    `(Here!) <http://www.diydrones.com/profiles/blogs/another-way-to-set-modes-on-turnigy-9x>`__ - `Turnigy 9x with ER9x firmware <http://diydrones.com/profiles/blogs/mode-switch-setup-for-turnigy>`__
 -  `Turnigy 9XR <http://diydrones.com/profiles/blogs/change-between-6-modes-with-turnigy-9xr-using-mixing>`__
+-  `Turnigy TGY-I6 <http://diydrones.com/profiles/blogs/flysky-fs-i6-flight-modes>`__

```

Comment:
@dtjw315 This (Turnigy TGY-I6) has the same URL as line 144 above (FlySky FS-I6) - is that correct?